public/app/plugins/datasource/jaeger/dependencyGraphTransform.ts

```diff
@@ -1,125 +0,0 @@
-import {
-  DataFrame,
-  DataQueryResponse,
-  FieldType,
-  MutableDataFrame,
-  NodeGraphDataFrameFieldNames as Fields,
-} from '@grafana/data';
-
-import { JaegerServiceDependency } from './types';
-
-interface Node {
-  [Fields.id]: string;
-  [Fields.title]: string;
-}
-
-interface Edge {
-  [Fields.id]: string;
-  [Fields.target]: string;
-  [Fields.source]: string;
-  [Fields.mainStat]: number;
-}
-
-/**
- * Error schema used by the Jaeger dependencies API.
- */
-interface JaegerDependenciesResponseError {
-  code: number;
-  msg: string;
-}
-
-interface JaegerDependenciesResponse {
-  data?: {
-    errors?: JaegerDependenciesResponseError[];
-    data?: JaegerServiceDependency[];
-  };
-}
-
-/**
- * Transforms a Jaeger dependencies API response to a Grafana {@link DataQueryResponse}.
- * @param response Raw response data from the API proxy.
- */
-export function mapJaegerDependenciesResponse(response: JaegerDependenciesResponse): DataQueryResponse {
```

Comment:
Code search shows this is not used.

<img width="801" height="366" alt="Image" src="https://github.com/user-attachments/assets/a851f6e7-875a-4842-8a81-a136761b96bb" />